Sources/RepoToolsCore/RepoActions.swift

```diff
@@ -25,6 +25,7 @@ public struct FetchOptions {
     public let url: String
     public let trace: Bool
     public let subDir: String?
+    public let fileHandler: String?
```

Comment:
Hrm, I changed the name of this in the bazel side to `extraction handler`, may want to switch it here too, thoughts?

Reply:
Seems reasonable to keep it consistent. Any reason to not name this `extractionHandler` here?

Reply:
I will switch it, I had been using this name and didn't realize I had switched until I saw this and made the comment.  I will make the switch when updating the swift code with the other suggestions.

Reply:
Switched this to `extractionHandler` to keep it in line with the naming in other changes.